triggers/run-completion-event-trigger/Makefile

```diff
@@ -0,0 +1,14 @@
+include ../../docker-targets.mk
+
+IMG := kfp-operator-run-completion-event-trigger
+
+build:
+	CGO_ENABLED=0 GOOS=$(GOOS) GOARCH=$(GOARCH) go build -o bin/run-completion-event-trigger ./main.go
+
+test:
+	go test ./... -tags=unit
+
+functional-test: docker-build
+	docker compose up -d
```

Comment:
```suggestion
functional-test:
        docker-build
	docker compose up -d
```

Reply:
I think as the docker-build is a direct call to another makefile definition as opposed to a CLI program it needs to be on the first line